File: app/src/main/java/com/stupidtree/hichat/HiApplication.java
package com.stupidtree.hichat;

import android.annotation.SuppressLint;
import android.app.Application;
import android.content.Context;

/**
 * APP类
 */
public class HiApplication extends Application {

    @SuppressLint("StaticFieldLeak")
    private static Context context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = getApplicationContext();
    }

    public static Context getContext() {
        return context;
    }
}

File: app/src/main/java/com/stupidtree/cloudliter/HiApplication.kt
package com.stupidtree.cloudliter

import android.annotation.SuppressLint
import android.app.Application
import android.content.Context

/**
 * APP类
 */
class HiApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        context = applicationContext
    }

    companion object {
        @SuppressLint("StaticFieldLeak")
        var context: Context? = null
            private set
    }
}